jetty-home/src/main/resources/modules/demo-jaas.mod

```diff
@@ -0,0 +1,17 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Setup for jaas demos.
+
+[environment]
+core
+
+[tags]
+demo
+webapp
+
+[depends]
+core-deploy
+
+[files]
+basehome:modules/demo.d/demo-login.conf|etc/demo-login.conf
```

Comment:
This line was removed from the eeX versions?

Reply:
Exactly. As there can only be a single config file.